client/web/compose/src/components/PageBlocks/Navigation/Configurator.vue

```diff
@@ -0,0 +1,300 @@
+            @click="handleAddButton"
+          >
+            <font-awesome-icon
+              :icon="['fas', 'plus']"
+              size="sm"
+              class="mr-1"
+            />
+            {{ $t("navigation.add") }}
+          </b-button>
+        </div>
+
+        <div class="mt-3">
+          <draggable
+            v-model="block.options.lists"
+            group="sort"
+            handle=".grab"
+          >
+            <div
+              v-for="(column, index) in block.options.lists"
+              :key="`column-${index}`"
+            >
+              <hr v-if="index" />
+
+              <table
+                class="
+                  table table-sm table-borderless table-responsive-lg
+                  nav-display-options-lists
+                "
+              >
+                <thead>
+                  <tr>
+                    <th style="width: auto" />
+                    <th>
+                      {{ $t("navigation.type") }}
+                    </th>
+                    <th>
+                      {{ $t("navigation.color") }}
+                    </th>
+                    <th>
+                      {{ $t("navigation.background") }}
+                    </th>
+                    <th class="text-center">
+                      {{ $t("navigation.enabled") }}
+                    </th>
+                    <th style="width: auto;min-width: 100px" />
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td class="align-middle">
+                      <font-awesome-icon
+                        :icon="['fas', 'bars']"
+                        class="grab text-grey"
+                      />
+                    </td>
+                    <td>
+                      <b-form-group>
+                        <b-form-select
+                          v-model="column.navigationType"
+                          :options="navigationItemTypes"
+                          size="sm"
+                        >
+                          <template #first>
+                            <b-form-select-option :value="''">
+                              {{ $t("navigation.none") }}
+                            </b-form-select-option>
+                          </template>
+                        </b-form-select>
+                      </b-form-group>
+                    </td>
+                    <td>
+                      <b-form-group>
+                        <b-form-input
+                          v-model="column.options.textColor"
+                          size="sm"
+                          type="color"
+                          debounce="300"
+                          class="w-100"
+                        />
+                      </b-form-group>
+                    </td>
+                    <td>
+                      <b-form-group>
+                        <b-form-select
+                          v-model="column.options.bgVariant"
```

Comment:
If you look at metric we have text color and background color, might make sense to do that here aswell to just keep it consistent. Sorry for late change